### DISCUSSION TITLE
no automatic opening of scripts when opening scenes

### DISCUSSION THREAD

**Discussion by AndHalfAFish** [godotengine/godot-proposals#4303] [Editor]
When you open an existing scene with a script attached to its root node, the script is opened automatically, and it is also added to the list of opened scripts. This quickly fills up the list of opened scripts, which reduces its usability  (lots of scripts added that are of no interest).  Also, often when I open a scene, it is to see the node hierarchy, not to work on the script.

**Comment by Mickeon** [comment]
You'll be happy to know you can disable this behaviour already:
![image](https://user-images.githubusercontent.com/66727710/160405480-b393464a-bebd-4832-b3e9-be688909b68a.png)

As to why it is enabled by default, my guess is that it helps understanding the correlation between Scenes and Scripts. which it definitely did for me when starting off.

**Reply by AndHalfAFish** [reply]
Great ! Thanks for mentioning.
